sway-fmt/src/fmt.rs

```diff
@@ -341,25 +341,25 @@ struct Rgb {
     blue: u64,
 }
 
-struct Structure {
+struct Bimbam {
```

Comment:
these renames are not needed

Reply:
forgot to remove aswell after changing the approach